go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/smartcontractkit/chainlink
 
 go 1.18
 
+replace github.com/smartcontractkit/libocr => github.com/jmank88/libocr v0.0.0-20220727210803-2d3649ef2507
```

Comment:
We want to flag this as temp?

Reply:
Yeah, didn't plan to merge until official support lands later this week.